### DISCUSSION TITLE
Unverständliche Preise

### DISCUSSION THREAD

**Discussion by Rellikon83** [butterwecksolutions/Spotmarket-Switcher#179] [General]
Hi,

ich bekomme bei den Total Charging Costs 0.1369 EUR ausgeworfen. Das ist seltsam. Er müsste in der Konsequenz laden, denn günstiger würde es garantiert nicht mehr. Der aktuelle Tibber Preis liegt bei 0.241 EUR und netto bei 0.079 EUR.

In der Config steht:

vat_rate=0.19
energy_fee=16.2
feedin_price=11.15
battery_lifecycle_costs_cent_per_kwh=4.0
energy_loss_percent=23.3
Economy Check = 1

> I: Bash Version: GNU bash, version 5.0.18(1)-release (arm-ve-linux-gnueabi)
I: Spotmarket-Switcher - Version 2.4.12
 [\]  Loading...Config validation passed.
I: Config or Controller was changed within the last 60 minutes. Cleaning /tmp directory.
I: Fetching today-data data from Tibber.
I: Please be patient. First we wait 25 seconds in case the system clock is not syncronized and not to overload the API.
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  4604  100  4442  100   162   8180    298 --:--:-- --:--:-- --:--:--  8494
Data available for 48 hours.
I: Please be patient. A delay of 10 seconds will help avoid overloading the Solarweather-API.
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   169  100   169    0     0    218      0 --:--:-- --:--:-- --:--:--   218
I: Please verify correct system time and timezone:
   Sun Jan 14 21:16:17 CET 2024
I: Current price is 0.0786 EUR/kWh energy price.
I: The average price will be 0.0813229 EUR/kWh energy price.
I: Highest price will be 0.1128 EUR/kWh energy price.
I: Sorted prices: 1:0.062 2:0.062 3:0.0621 4:0.0629 5:0.0639 6:0.064 7:0.064 8:0.0647 9:0.065 10:0.0652 11:0.0672 12:0.0673
                  13:0.0679 14:0.0692 15:0.0717 16:0.0754 17:0.0767 18:0.077 19:0.0786 20:0.0792 21:0.08 22:0.0804 23:0.0806 24:0.081
                  25:0.0811 26:0.0818 27:0.0822 28:0.0823 29:0.0838 30:0.0839 31:0.0845 32:0.0864 33:0.0866 34:0.0889 35:0.0894 36:0.0895
                  37:0.0903 38:0.0918 39:0.0919 40:0.0949 41:0.097 42:0.0975 43:0.0993 44:0.1003 45:0.1024 46:0.1065 47:0.1104 48:0.1128
I: Charge at prices: 1 2 3 4 5 6
I: Dynamic ESS discharge (depending SOC) at prices: 42 43 44 45 46 47 48
I: Sunrise today will be 08:09 and sunset will be 16:44. Suntime will be 515 minutes.
I: Solarenergy today will be 1.7 megajoule per sqaremeter with 1.7 percent clouds. The temperature is -2.9°C with 0.1cm snowdepth.
I: Solarenergy tomorrow will be 3.1 megajoule per squaremeter with 3.1 percent clouds. The temperature will be -3.1°C with 1.3cm snowdepth.
I: At 1.7 megajoule there will be a dynamic SOC charge-target of 82% calculated. The rest is reserved for solar.
I: Victron scheduled charging is OFF. Charging was not executed. Total charging costs: 0.1369€
I: Victron discharging (ESS) is OFF. Battery SOC is at 17%.
D: skip Fritz DECT. not activated
D: skip Shelly Api. not activated

> 

Muss man an der Preis Matrix noch rumschrauben? Da fehlt es etwas an Doku um das zu verstehen. Aktuell ist das entladen gestoppt, aber ich sehe nicht wann wer wieder entläd obwohl in der Batterie noch etwas drin ist.
Verstehe ich es ungefähr richtig: Wenn ich in der Matrix unter Minimum Soc überall 0 eintragen würde, dann würde er unabhängig der Preise entladen. Ansonsten entläd er nur wenn sowohl aktueller Soc als auch Preis zueinander passt? In meinem Fall 17% und der Preis liegt aktuell bei 7,9 Cent. In der Matrix steht bei 16% Soc ein Preis von 9,75 Cent. Heißt er entläd nicht, da der aktuelle Preis unter dem zugeordneten Soc Preis liegt?

**Comment by butterwecksolutions** [comment]
> Hi,
> 
> ich bekomme bei den Total Charging Costs 0.1369 EUR ausgeworfen. Das ist seltsam. Er müsste in der Konsequenz laden, denn günstiger würde es garantiert nicht mehr. Der aktuelle Tibber Preis liegt bei 0.241 EUR und netto bei 0.079 EUR.
> 
> In der Config steht:
> 
> vat_rate=0.19 energy_fee=16.2 feedin_price=11.15 battery_lifecycle_costs_cent_per_kwh=4.0 energy_loss_percent=23.3 Economy Check = 1
> 
> > I: Bash Version: GNU bash, version 5.0.18(1)-release (arm-ve-linux-gnueabi)
> > I: Spotmarket-Switcher - Version 2.4.12
> > []  Loading...Config validation passed.
> > I: Config or Controller was changed within the last 60 minutes. Cleaning /tmp directory.
> > I: Fetching today-data data from Tibber.
> > I: Please be patient. First we wait 25 seconds in case the system clock is not syncronized and not to overload the API.
> > % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
> > Dload  Upload   Total   Spent    Left  Speed
> > 100  4604  100  4442  100   162   8180    298 --:--:-- --:--:-- --:--:--  8494
> > Data available for 48 hours.
> > I: Please be patient. A delay of 10 seconds will help avoid overloading the Solarweather-API.
> > % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
> > Dload  Upload   Total   Spent    Left  Speed
> > 100   169  100   169    0     0    218      0 --:--:-- --:--:-- --:--:--   218
> > I: Please verify correct system time and timezone:
> > Sun Jan 14 21:16:17 CET 2024
> > I: Current price is 0.0786 EUR/kWh energy price.
> > I: The average price will be 0.0813229 EUR/kWh energy price.
> > I: Highest price will be 0.1128 EUR/kWh energy price.
> > I: Sorted prices: 1:0.062 2:0.062 3:0.0621 4:0.0629 5:0.0639 6:0.064 7:0.064 8:0.0647 9:0.065 10:0.0652 11:0.0672 12:0.0673
> > 13:0.0679 14:0.0692 15:0.0717 16:0.0754 17:0.0767 18:0.077 19:0.0786 20:0.0792 21:0.08 22:0.0804 23:0.0806 24:0.081
> > 25:0.0811 26:0.0818 27:0.0822 28:0.0823 29:0.0838 30:0.0839 31:0.0845 32:0.0864 33:0.0866 34:0.0889 35:0.0894 36:0.0895
> > 37:0.0903 38:0.0918 39:0.0919 40:0.0949 41:0.097 42:0.0975 43:0.0993 44:0.1003 45:0.1024 46:0.1065 47:0.1104 48:0.1128
> > I: Charge at prices: 1 2 3 4 5 6
> > I: Dynamic ESS discharge (depending SOC) at prices: 42 43 44 45 46 47 48
> > I: Sunrise today will be 08:09 and sunset will be 16:44. Suntime will be 515 minutes.
> > I: Solarenergy today will be 1.7 megajoule per sqaremeter with 1.7 percent clouds. The temperature is -2.9°C with 0.1cm snowdepth.
> > I: Solarenergy tomorrow will be 3.1 megajoule per squaremeter with 3.1 percent clouds. The temperature will be -3.1°C with 1.3cm snowdepth.
> > I: At 1.7 megajoule there will be a dynamic SOC charge-target of 82% calculated. The rest is reserved for solar.
> > I: Victron scheduled charging is OFF. Charging was not executed. Total charging costs: 0.1369€
> > I: Victron discharging (ESS) is OFF. Battery SOC is at 17%.
> > D: skip Fritz DECT. not activated
> > D: skip Shelly Api. not activated
> 
> > 
> 
> Muss man an der Preis Matrix noch rumschrauben? Da fehlt es etwas an Doku um das zu verstehen. Aktuell ist das entladen gestoppt, aber ich sehe nicht wann wer wieder entläd obwohl in der Batterie noch etwas drin ist. Verstehe ich es ungefähr richtig: Wenn ich in der Matrix unter Minimum Soc überall 0 eintragen würde, dann würde er unabhängig der Preise entladen. Ansonsten entläd er nur wenn sowohl aktueller Soc als auch Preis zueinander passt? In meinem Fall 17% und der Preis liegt aktuell bei 7,9 Cent. In der Matrix steht bei 16% Soc ein Preis von 9,75 Cent. Heißt er entläd nicht, da der aktuelle Preis unter dem zugeordneten Soc Preis liegt?

Moin,
warum sollte geladen werden wenn der Gesamtpreis incl. Batteriekosten und Ladeverluste höher als der höchste Tagespreis ist?
Die letzten Tage macht es keinen Sinn zu laden, denn die Preiskurven sind zu flach, da kann man den Strom auch ohne Ladeverluste direkt verbrauchen.

Du hast außerdem noch nicht auf Gesamtpreise umkonfiguriert, sonst würde noch etwas genauer gerechnet.
Steht ja auch da: Current price is 0.0786 EUR/kWh **energy price.**, Highest price will be 0.1128 EUR/kWh **energy price.**

Gruß!
Christian

**Comment by Rellikon83** [comment]
@christian1980nrw danke das habe ich übersehen. Jetzt macht der Output deutlich mehr Sinn.

Ich habe jedoch ein neues Problem. Er läd die Batterien aktuell nicht nach. Victron steht auf Bulk und die Batterien auf Idle. ESS#7 (Limit Inverter). Limit heißt doch aber nicht, dass er nicht mehr laden dürfte. Stellt Ihr noch was ein?

**Comment by butterwecksolutions** [comment]
Ess7 bedeutet das auf höhere Preise gewartet wird bis entladen wird. Die Inverterleistung wurde dazu auf 0W gestellt.
Falls Dein Akku leer ist, wird das vermutlich nur eine Preisstunde am Tag sein wo entladen wird.
Je voller der Akku desto mehr Preisstunden werden aktiviert.

Gruß!
Christian

**Reply by Rellikon83** [reply]
Sehr ausgeklügelt. Klingt logisch. 
kannst du noch etwas sagen weshalb der Akku nicht über den sonnenüberschuss geladen wird?

**Comment by butterwecksolutions** [comment]
Doch ...das ESS funktioniert ansonsten wie immer. Der Akku wird von Solar geladen aber die Entladung ist ggf. gesperrt.

**Reply by Rellikon83** [reply]
Hab’s jetzt mal beobachtet. Das ess läuft super in Kombination mit tibber. Die entladezeiten werden schön auf die hochpreisigen Zeiten gelegt. Die Matrix arbeitet sauber. Nachladekosten sind zu hoch (0.31 Euro), so dass ein nachladen der Batterie vermutlich seltener wird (find ich gar nicht so schlecht um sie zu schonen). Positiver Nebeneffekt: in den günstigen Zeiten wird das Auto geladen. Gerade dort gibt’s oft den entladestop und der Akku wird nicht leer gesaugt. Bisher Good Job !
